CryptKeeperZK/crypt-keeper-extension
# feat: derive import identities from seed phrase

## Explanation

This PR adds support for importing identities and derive them from user's seed phrase.

Details are below:
- [x] Add signature options for identity import
- [x] Minor refactoring

## Related Issues

Blocked by #1040 
Related to #906

## Screenshots

<details>
<summary>Expand</summary>

![image](https://github.com/CryptKeeperZK/crypt-keeper-extension/assets/14254374/2c93e4c3-4701-41bf-aef3-f9c9cd007579)

</details>

## Manual Testing Steps

1. Try to import identity from demo, connect, create identity screens.
2. Check cryptkeeper sign
3. Check metamask sign with and without connect

## Pre-Merge Checklist

### Assignees Checklist


### Reviewers Checklist

- [x] Manual testing checked and passed.
